data/hlint.yaml

```diff
@@ -296,6 +296,10 @@
 
     - warn: {lhs: \x -> x, rhs: id}
     - warn: {lhs: \x y -> x, rhs: const}
+    - warn: {lhs: curry fst, rhs: const}
+    - warn: {lhs: \x y -> y, rhs: const id}
```

Comment:
I find the lambda form more explicit and clearer. I think I'd probably even substitute the other way in my code...

Reply:
I removed this one, since unlike `curry snd` and `flip const`, it is just as efficient as `const id`.